packages/core/src/dynamoDb/commands/searchDynamoDbTables.ts

```diff
@@ -0,0 +1,110 @@
+/*!
```

Comment:
Could probably just rename this file to searchTables instead since the folder is already in dynamoDB